pages/common/zdiff.md

```diff
@@ -0,0 +1,12 @@
+# zdiff
+
+> Invoke `diff` on gzipped files.
+> More information: <https://manned.org/zdiff>.
+
+- Compare two files, uncompressing them if necessary:
+
+`zdiff {{path/to/file1.txt.gz}} {{path/to/file2.txt.gz}}`
+
+- Invoke `diff` on `path/to/file.txt` and `path/to/file.txt.gz` (which is assumed to exist):
```

Comment:
```suggestion
- Invoke `diff` on `path/to/file.txt` and `path/to/file.txt.gz` (assuming they exist):
```
I am not sure about this command description. Let me ask someone.

Reply:
```suggestion
- Compare a file with a gzipped archive with the same name:
```